sources/Cargo.toml

```diff
@@ -206,6 +208,8 @@ unindent = "0.2"
 url = "2"
 walkdir = "2.4"
 which = "4"
+zbus = { version = "5.11", features = ["p2p"] }
+zvariant = "5.7"
```

Comment:
Let's do another pass over these dependencies and make sure we disable any features we don't actually need.